src/core/public/doc_links/doc_links_service.ts

```diff
@@ -46,10 +46,84 @@ export class DocLinksService {
     const OPENSEARCH_DOCS = `https://opensearch.org/docs/opensearch/`;
     const OPENSEARCH_DASHBOARDS_DOCS = `https://opensearch.org/docs/dashboards/`;
 
+    const OPENSEARCH_VERSIONED_DOCS =
+      OPENSEARCH_WEBSITE_URL + `docs/` + DOC_LINK_VERSION + `/opensearch/`;
+    const OPENSEARCH_DASHBOARDS_VERSIONED_DOCS =
+      OPENSEARCH_WEBSITE_URL + `docs/` + DOC_LINK_VERSION + `/dashboards/`;
+
     return deepFreeze({
       DOC_LINK_VERSION,
       OPENSEARCH_WEBSITE_URL,
       links: {
+        opensearch: {
```

Comment:
I like the organization here based on the documentation website. My initial thought is to bring in the other links that exist into these objects to separate the documentation that exists from the links that don't exist yet. 

For example, we could nest the `aggs` links under `opensearch.aggregations[metric/bucket/pipeline]` and have something like a `root` or `base` property that links to the root page without any anchor. 

```
opensearch: {
  ...
  aggregations: {
    metric: {
      root: `${OPENSEARCH_VERSIONED_DOCS}metric-agg`,
      avg: `${OPENSEARCH_VERSIONED_DOCS}metric-agg/#sum-min-max-avg`,
      cardinality: `${OPENSEARCH_VERSIONED_DOCS}metric-agg/#cardinality`,
      ...
    },
    ...
  },
  ...
},
opensearchDashboards: {
  ...
},
noDocumentation: {
  auditbeat: {
    ...
  },
  filebeat: {
    ...
  },
  ...
}
```

Reply:
Good point. My initial thought is to void  3 level+ nested structure so that the file looks more cleaner. (Only a few anchor is used in codebase.) But i am open to make the change.